### PR TITLE
Created requirements.txt and added installation instructions

### PR DESCRIPTION
**Pull request by asclines**
For future reference developers, when you add new modules update the `requirements.txt` using Method 2 from [here](http://www.idiotinside.com/2015/05/10/python-auto-generate-requirements-txt/).

### CODE REVIEW

**Review by dlowrey — COMMENTED** [review]
in the requirements.txt, I am using the `mysql-connector-python (2.1.5)`, do these need to match?

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mysql_connector_repackaged==0.3.1
```
Comment (dlowrey) [inline requirements.txt]: I am using `mysql-connector-python (2.1.5)`, do these need to match?

Comment (asclines) [reply requirements.txt]: Yes, and I am working on that. Problem is [PyPi](https://pypi.python.org/pypi/mysql-connector) only has up to 2.1.4

Comment (asclines) [reply requirements.txt]: Update, got it working. The hard way.

**Review by dlowrey — APPROVED** [review]
Looks good

**Pull request merged**